### ISSUE TITLE
Missing Japanese Translations [Fast_Htmlcs]

### ISSUE THREAD

**Issue by j-mendez** [a11ywatch/kayle#4]
The following codes have missing 'ja' translations.

WCAG2AA.Principle1.Guideline1_3.1_3_4.
WCAG2AA.Principle1.Guideline1_4.1_4_10.C32,C31,C33,C38,SCR34,G206
WCAG2AA.Principle1.Guideline1_4.1_4_11.G195,G207,G18,G145,G174,F78
WCAG2AA.Principle1.Guideline1_4.1_4_12.C36,C35
WCAG2AA.Principle1.Guideline1_4.1_4_13.F95
WCAG2AA.Principle2.Guideline2_1.2_1_4.
WCAG2AA.Principle2.Guideline2_5.2_5_1.
WCAG2AA.Principle2.Guideline2_5.2_5_2.
WCAG2AA.Principle2.Guideline2_5.2_5_3.F96
WCAG2AA.Principle2.Guideline2_5.2_5_4.
WCAG2AA.Principle4.Guideline4_1.4_1_3.
WCAG2AA.Principle1.Guideline1_3.1_3_5.H98

**Comment by j-mendez** [comment]
completed awhile back
